tools/devops/automation/scripts/Artifacts.Tests.ps1

```diff
@@ -0,0 +1,64 @@
+Using module ./Artifacts.psm1
+
+Describe "Artifacts tests" {
+
+    BeforeAll {
+        $dataPath = Join-Path -Path $PSScriptRoot -ChildPath "test_data" 
+        $dataPath = Join-Path -Path $dataPath -ChildPath "artifacts.json"
+        $data = Get-Content -Path $dataPath | ConvertFrom-Json
+    }
+
+    It "is correctly create" {
```

Comment:
I have not seen ruby style tests like this before. Is this a common powershell thing?